arbnode/transaction_streamer.go

```diff
@@ -308,7 +308,13 @@ func (s *TransactionStreamer) reorg(batch ethdb.Batch, count arbutil.MessageInde
 					continue
 				}
 				msgBlockNum := new(big.Int).SetUint64(oldMessage.Message.Header.BlockNumber)
-				delayedInBlock, err := s.delayedBridge.LookupMessagesInRange(s.GetContext(), msgBlockNum, msgBlockNum, nil)
+				// If not started, use a background context.
+				// This can happening when reorging on startup using the init flag.
+				ctx := context.Background()
```

Comment:
let's do that at the start of the function.. same context might be used later.. e.g. in s.exec.Reorg (line 340/346) when this becomes an async function.

Reply:
I'm also thinking of ways to send the context to reorg message, to make it clear when it can and when it cannot happen before start.. but I'm not sure there is a good way to do so that's also simple enough.